### ISSUE TITLE
How to get a Map[String, List[String]]] where Keys have reserved tokens like '.' or '$'

### ISSUE THREAD

**Issue by JayChandler** [kxbmap/configs#7]
I have this simple config

<pre>
dtool {
	config {
		partners {
			"nic.at" =  ["\\.at$"]
			
		}
	}
}
</pre>

When i try to run this code in my app
```java
appConfig.get[Map[String, List[String]]]("dtool.config.partners")
```
I get an exception

**Exception in thread "main" com.typesafe.config.ConfigException$Missing: No configuration setting found for key 'nic'**

When i try to double quote **""nic.at""** i get this error

**ConfigException$WrongType: : nic has type OBJECT rather than LIST**

All i want is to get a **Map[String, List[String]]]**

I also looked up the typesafe documentation but could not find any answer

**Comment by kxbmap** [comment]
I published 0.2.5-SNAPSHOT. Could you confirm that it fixes this issue?

**Comment by JayChandler** [comment]
Hi

i get this message

**java.lang.NoClassDefFoundError: com/github/kxbmap/configs/package$EnrichTypesafeConfig$**. Am i missing somehting ?

**Comment by JayChandler** [comment]
This is working. Got following output
<pre>
Map(\.at$ -> nic.at)
</pre>

Can u publish a maven repo?

**Comment by kxbmap** [comment]
Thanks for confirming.
Just published 0.2.5 and it will sync to maven in few hours.

**Comment by JayChandler** [comment]
Thanks for fixing this so fast
